### ISSUE TITLE
unresolved import `gradient::CGGradientDrawingOptions`

### ISSUE THREAD

**Issue by kvark** [servo/core-foundation-rs#393]
See [travis log](https://travis-ci.org/github/gfx-rs/gfx/jobs/702707478):
```
error[E0432]: unresolved import `gradient::CGGradientDrawingOptions`

  --> /Users/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/core-graphics-0.19.1/src/context.rs:15:28

   |

15 | use gradient::{CGGradient, CGGradientDrawingOptions};

   |                            ^^^^^^^^^^^^^^^^^^^^^^^^ no `CGGradientDrawingOptions` in `gradient`

error: cannot find macro `bitflags` in this scope

  --> /Users/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/core-graphics-0.19.1/src/gradient.rs:22:1

   |

22 | bitflags! {

   | ^^^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0432`.

error: could not compile `core-graphics`.
```

**Comment by jdm** [comment]
How are you receiving this error? `cargo build` on macOS in the core-graphics subdirectory builds successfully for me.

**Comment by jdm** [comment]
Oh, this might be the same issue that was just fixed in https://github.com/servo/core-foundation-rs/pull/392, if you're building for iOS.

**Comment by jdm** [comment]
Published 0.19.2.
